lib/index.js

```diff
@@ -33,10 +34,15 @@ co(function * () {
 		fly.emit('fly_run', {path: fly.file}).start(tasks)
 	}
 }).catch(function (e) {
-	if (e.code === 'ENOENT') {
-		utils.error('No Flyfile? See the Quickstart guide → git.io/fly-quick')
```

Comment:
I know it's seemingly silly, but we probably _should_ still keep the no-flyfile error. Useful for people who are converting and/or never read the README